src/Popper.js

```diff
@@ -64,13 +67,24 @@ export class InnerPopper extends React.Component<PopperProps, PopperState> {
 
   state = {
     data: undefined,
+    placement: undefined,
+    lastPropsPlacement: undefined,
```

Comment:
maybe `previousPlacement` a la `previousProps`?

Reply:
Why do we need to keep track of the previous placement in the state? Doesn't `prevProps` already provide this info?

Reply:
prevProps doesn't exist in the new lifestyle hook